src/components/PopoverMenu/index.js

```diff
@@ -1,135 +1,99 @@
 
-    resetFocusAndHideModal() {
-        this.setState({focusedIndex: -1}); // Reset the focusedIndex on modal hide
-        this.removeKeyboardListener();
-        if (this.selectedItem) {
-            this.selectedItem.onSelected();
-            this.selectedItem = null;
-        }
-    }
+    const [focusedIndex, setFocusedIndex] = useArrowKeyFocusManager({initialFocusedIndex: -1, maxIndex: props.menuItems.length - 1});
+    useKeyboardShortcut(
+        CONST.KEYBOARD_SHORTCUTS.ENTER,
+        () => {
+            if (focusedIndex === -1) {
+                return;
+            }
+            selectItem(focusedIndex);
+            setFocusedIndex(-1); // Reset the focusedIndex on selecting any menu
+        },
+        {isActive: props.isVisible},
+    );
 
-    render() {
-        return (
-            <Popover
-                anchorPosition={this.props.anchorPosition}
-                onClose={this.props.onClose}
-                isVisible={this.props.isVisible}
-                onModalHide={this.resetFocusAndHideModal}
-                animationIn={this.props.animationIn}
-                animationOut={this.props.animationOut}
-                animationInTiming={this.props.animationInTiming}
-                disableAnimation={this.props.disableAnimation}
-                fromSidebarMediumScreen={this.props.fromSidebarMediumScreen}
-            >
-                <View style={this.props.isSmallScreenWidth ? {} : styles.createMenuContainer}>
-                    {!_.isEmpty(this.props.headerText) && (
-                        <View style={styles.createMenuItem}>
-                            <Text style={[styles.createMenuHeaderText, styles.ml3]}>{this.props.headerText}</Text>
-                        </View>
-                    )}
-                    <ArrowKeyFocusManager
-                        focusedIndex={this.state.focusedIndex}
-                        maxIndex={this.props.menuItems.length - 1}
-                        onFocusedIndexChanged={(index) => this.setState({focusedIndex: index})}
-                    >
-                        {_.map(this.props.menuItems, (item, menuIndex) => (
-                            <MenuItem
-                                key={item.text}
-                                icon={item.icon}
-                                iconWidth={item.iconWidth}
-                                iconHeight={item.iconHeight}
-                                title={item.text}
-                                description={item.description}
-                                onPress={() => this.selectItem(item)}
-                                focused={this.state.focusedIndex === menuIndex}
-                            />
-                        ))}
-                    </ArrowKeyFocusManager>
-                </View>
-            </Popover>
-        );
-    }
-}
+    return (
+        <PopoverWithMeasuredContent
+            anchorPosition={props.anchorPosition}
+            anchorAlignment={props.anchorOrigin}
```

Comment:
This PR caused a regression https://github.com/Expensify/App/issues/19453
We are passing the wrong prop here anchorAlignment={props.anchorOrigin} should be anchorAlignment={props.anchorAlignment} More detail https://github.com/Expensify/App/issues/19453#issuecomment-1560148262